### PR TITLE
fix updated_at when updating data

### PR DESCRIPTION
**Pull request by merlouduron**
# Pull Request Review Comments

## Overview
- **Purpose of the PR**:
    - [ ] Feature addition
    - [ ] Bug fix
    - [ ] Refactoring
    - [ ] Documentation update
    - [ ] Other (please specify below)

## Specific Areas for Verification
- Please specify any particular requests or questions for the reviewer here.

## Suggestions/Concerns
- If you have any suggestions or concerns, please mention them here.

---

_**Additional Comments for the Reviewer (Optional)**_

- Add any additional comments or information for the reviewer here.

### CODE REVIEW

File: client/src/pages/Expenses/ExpensesListAndEdit.tsx [hunk 1]
```diff
@@ -99,14 +97,46 @@ const ExpensesList: React.FC = () => {
     const updatedData = [...combinedData]
     updatedData[index] = {
       ...updatedData[index],
-      [name]: value,
+      [name]: value, 
     }
     setExpensesList(updatedData)
   }
 
+
```
Comment (edward-stalley-work) [inline client/src/pages/Expenses/ExpensesListAndEdit.tsx:105]: we can delete this space

File: client/src/pages/Expenses/ExpensesListAndEdit.tsx [hunk 2]
```diff
@@ -99,14 +97,46 @@ const ExpensesList: React.FC = () => {
     const updatedData = [...combinedData]
     updatedData[index] = {
       ...updatedData[index],
-      [name]: value,
+      [name]: value,
```
Comment (edward-stalley-work) [inline client/src/pages/Expenses/ExpensesListAndEdit.tsx:100]: can delete this space that was added

File: client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx [hunk 3]
```diff
@@ -78,36 +78,57 @@ const BusinessDivisionsListAndEdit: React.FC = () => {
...
+            if (Object.keys(changes).length > 1) {
+              modifiedFields.push(changes)
+            }
+          }
+        })
+
+        return modifiedFields
+      }
+      const modifiedFields = getModifiedFields(originalBusiness, business)
+      console.log(modifiedFields)
```
Comment (edward-stalley-work) [inline client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx:120]: can delete console.log

File: client/src/pages/CostOfSales/CostOfSalesListAndEdit.tsx [hunk 4]
```diff
@@ -93,6 +90,39 @@ const CostOfSalesList: React.FC = () => {
...
+            if (hasChanges) {
+              modifiedFields.push(changes)
+            }
+          }
+        })
+        return modifiedFields
+      }
+
+      const modifiedFields = getModifiedFields(originalCostOfSales, validData)
+      console.log(modifiedFields)
```
Comment (edward-stalley-work) [inline client/src/pages/CostOfSales/CostOfSalesListAndEdit.tsx:121]: can delete console.log

**Review by edward-stalley-work — CHANGES_REQUESTED** [review]
Thank you merlou. Please see comments - just some spaces and console log removals.

File: client/src/pages/Expenses/ExpensesListAndEdit.tsx (hunk 1 above)
Comment (merlouduron) [reply client/src/pages/Expenses/ExpensesListAndEdit.tsx:105]: Fixed.

File: client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx (hunk 3 above)
Comment (merlouduron) [reply client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx:120]: Fixed.

File: client/src/pages/Expenses/ExpensesListAndEdit.tsx [hunk 5]
```diff
@@ -99,14 +97,46 @@ const ExpensesList: React.FC = () => {
...
+          if (hasChanges) {
+            modifiedFields.push(changes)
+          }
+        }
+      })
+      return modifiedFields
+    }
+
+    const modifiedFields = getModifiedFields(originalExpenseList, validData)
+    console.log(modifiedFields)
```
[the remark this answers is not in the document]
Comment (merlouduron) [reply client/src/pages/Expenses/ExpensesListAndEdit.tsx:136]: Fixed.

**Review by edward-stalley-work — APPROVED** [review]

**Pull request merged**